lasagne/layers/special.py

```diff
@@ -662,38 +688,46 @@ def get_output_for(self, inputs, **kwargs):
         # see eq. (1) and sec 3.1 in [1]
         # Get input and destination control points
         input, dest_offsets = inputs
-        return _transform_thin_thin_plate_spline(dest_offsets,
-                                                 input,
-                                                 self.source_points,
-                                                 self.L_inv,
-                                                 self.downsample_factor)
+        return _transform_thin_thin_plate_spline(
+                dest_offsets, input, self.right_mat, self.L_inv,
+                self.source_points, self.out_height, self.out_width,
+                self.precompute_grid, self.downsample_factor)
+
 
+def _transform_thin_thin_plate_spline(
+        dest_offsets, input, right_mat, L_inv, source_points, out_height,
+        out_width, precompute_grid, downsample_factor):
 
-def _transform_thin_thin_plate_spline(dest_offsets, input, source_points,
-                                      L_inv, downsample_factor):
     num_batch, num_channels, height, width = input.shape
     num_control_points = source_points.shape[1]
 
     # reshape destination offsets to be (num_batch, 2, num_control_points)
     # and add to source_points
     dest_points = source_points + T.reshape(
-            dest_offsets, (num_batch, 2, num_control_points))
+        dest_offsets, (num_batch, 2, num_control_points))
```

Comment:
I'd prefer double indent as before (make sure to use the same pep8 version as in `requirements-dev.txt`).